test/gaudi_opts/fccRec_e4h_input.py

```diff
@@ -86,7 +86,7 @@
 InitDD4hep.OutputLevel = WARNING
 InitDD4hep.ProcessorType = "InitializeDD4hep"
 InitDD4hep.Parameters = {
-                         "DD4hepXMLFile": [os.environ["LCGEO"]+"/FCCee/compact/FCCee_o1_v04/FCCee_o1_v04.xml"],
+                         "DD4hepXMLFile": [os.environ["LCGEO"]+"/FCCee/CLD/compact/FCCee_o1_v04/FCCee_o1_v04.xml"],
```

Comment:
```suggestion
                         "DD4hepXMLFile": [os.environ["K4GEO"]+"/FCCee/CLD/compact/FCCee_o1_v04/FCCee_o1_v04.xml"],
```
We have both in the environment but I guess we prefer now to use K4GEO since the name of the package was changed